app/views/public_health/exposure.html.erb

```diff
@@ -1,7 +1,7 @@
 <% content_for :assets do %>
-  <%= javascript_pack_tag "datatables" %>
-  <%= javascript_pack_tag "xss" %>
   <%= javascript_pack_tag "moment" %>
+  <%= javascript_pack_tag "reacttable" %>
+  <%= javascript_pack_tag "xss" %>
```

Comment:
I think we can totally dump this now! React/JSX handles what this was needed for in datatables.